control-plane/connect-inject/controllers/endpoints/endpoints_controller.go

```diff
@@ -209,7 +198,7 @@ func (r *EndpointsController) Reconcile(ctx context.Context, req ctrl.Request) (
 					continue
 				}
 
-				svcName, ok := pod.Annotations[annotationKubernetesService]
+				svcName, ok := pod.Annotations[common.AnnotationKubernetesService]
```

Comment:
We might want to consider an annotations package so that you could do something like:
 `svcName, ok := pod.Annotations[annotations.KubernetesService]` 
 vs 
 `svcName, ok := pod.Annotations[common.AnnotationKubernetesService]`

Reply:
I changed it to be `constants` based on our convo in Slack because that file has annotations and labels plus other random stuff. So it doesn't make sense to move it to annotations package